feature-libs/checkout/base/core/services/checkout-page-meta.resolver.ts

```diff
@@ -27,8 +26,6 @@ export class CheckoutPageMetaResolver
   extends PageMetaResolver
   implements PageTitleResolver, PageDescriptionResolver, PageRobotsResolver
```

Comment:
I missed this, but for convention on interfaces that helps to 'include' the abstract methods to be implemented, you should implement PageHeadingResolver